lifecycle-operator/controllers/common/evaluation/evaluationhandler_test.go

```diff
@@ -346,14 +340,7 @@ func TestEvaluationHandler_createEvaluation(t *testing.T) {
 		t.Run(tt.name, func(t *testing.T) {
 			err := v1alpha3.AddToScheme(scheme.Scheme)
 			require.Nil(t, err)
-			handler := EvaluationHandler{
-				SpanHandler: &kltfake.ISpanHandlerMock{},
-				Log:         ctrl.Log.WithName("controller"),
-				EventSender: NewK8sSender(record.NewFakeRecorder(100)),
-				Client:      fake.NewClientBuilder().Build(),
-				Tracer:      trace.NewNoopTracerProvider().Tracer("tracer"),
-				Scheme:      scheme.Scheme,
-			}
+			handler := NewEvaluationHandler(fake.NewClientBuilder().Build(), common.NewK8sSender(record.NewFakeRecorder(100)), ctrl.Log.WithName("controller"), trace.NewNoopTracerProvider().Tracer("tracer"), scheme.Scheme, &kltfake.ISpanHandlerMock{})
```

Comment:
please split this up so that this is not all in one line, this will make it a bit easier to read

Reply:
Alright, let me do that now